src/database/operations/load/loadBulkUploadBundle.ts

```diff
@@ -1,16 +1,14 @@
 import { isBulkUpload } from '../../../types';
 import { loadBundle } from './loadBundle';
 import type { TinyPgParams } from 'tinypg';
-import type {
-  Bundle,
-  BulkUpload,
-} from '../../../types';
+import type { Bundle, BulkUpload } from '../../../types';
 
 export const loadBulkUploadBundle = async (
-  queryParameters: TinyPgParams,
-): Promise<Bundle<BulkUpload>> => loadBundle(
-  'bulkUploads.selectWithPagination',
-  queryParameters,
-  'bulk_uploads',
-  isBulkUpload,
-);
+	queryParameters: TinyPgParams,
+): Promise<Bundle<BulkUpload>> =>
+	loadBundle(
```

Comment:
Oh, ew, here's another place where I disagree with Prettier. Should we start wrapping all multiline expressions in parens?

Reply:
Blegh that decision feels so odd to me!  I guess I can only hope I'll get used to it...